deploy/runtime/startup.lk

```diff
@@ -263,15 +263,38 @@ function setup_residential_pages( mode ) // mode determines whether or not the b
   setup_elec_load_page( mode );
 }
 
-function setup_thirdparty_pages( load ) // FOR NOW, THIRD PARTY OWNERSHIP MODEL IS ONLY VALID FOR PVWATTS AND PV DETAILED and generic system!!
+function setup_thirdparty_pages( load ) // Third party host
 {
   setting( { 'analysis_period_var'='cf_length' } );
   addpage( [[ 'Financial Third Party Ownership']],
-           { 'sidebar'='Financial Parameters', 'help'='fin_thirdparty'} );
+           { 'sidebar'='Financial Parameters', 'help'='fin_tpo_host'} );
```

Comment:
Thanks for breaking out the TPO models - any additional variables added or removed based on new set of stacked pages for patching?

Reply:
This is only an update to the 'help' parameter of addpage() so it points to the correct topic in Help. It does not change the UI pages.